mlir/include/mlir/InitAllDialects.h

```diff
@@ -14,6 +14,7 @@
 #ifndef MLIR_INITALLDIALECTS_H_
 #define MLIR_INITALLDIALECTS_H_
 
+#include "mlir/Config/mlir-config.h"
```

Comment:
Why is it needed here?

Reply:
Was needed when PDL was optionally built, removed.